sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_receiver_async.py

```diff
@@ -69,7 +72,7 @@ async def _incoming_transfer(self, frame):
                 self._received_payload = bytearray()
             else:
                 message = decode_payload(frame[11])
-            delivery_state = await self._process_incoming_message(frame, message)
+            delivery_state = await self._process_incoming_message(self._first_frame, message)
             if not frame[4] and delivery_state:  # settled
                 await self._outgoing_disposition(
                     first=frame[1],
```

Comment:
I'm not sure if we hit this condition during our debugging, but we might have to change this to `self._first_frame[1]` to get the delivery id, else we end up sending a None on multi transfers.

Reply:
ah sorry, wrong lined selected. I meant line 78, first=frame[1]